code/modules/mob/living/say.dm

```diff
@@ -406,7 +406,7 @@ var/list/department_radio_keys = list(
 		if(M.client)
 			speech_bubble_recipients.Add(M.client)
 	spawn(0)
-		flick_overlay(image('icons/mob/talk.dmi', src, "h[bubble_type][say_test(message)]",MOB_LAYER+1), speech_bubble_recipients, 30)
+		flick_overlay(image('icons/mob/talk.dmi', isturf(loc) ? src : loc, "h[bubble_type][say_test(message)]",MOB_LAYER+1), speech_bubble_recipients, 30)
```

Comment:
You mean instead of loc? We want it to be parented to loc, that way it will follow mechs or the locker if it's being pulled or the alium queen that ate you or whatever.